packages/core/less/login-page.less

```diff
@@ -0,0 +1,42 @@
+.login_card_input {
+  min-height: 61px;
+  margin-bottom: 5px;
+}
+
+.login-pf {
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  padding: 20px;
+}
+
+.login-form-error {
+  margin-bottom: 15px;
+  color: @color-pf-red;
+}
+
+.login-pf-brand {
+  margin-top: 0 !important;
+}
```

Comment:
I don't think we should change this. PF supplies the desired margin top. The issue is only when there is an inline notification above that this makes sense. Could we modify the inline notification to be positioned within this margin? Currently when you dismiss the notification the page jumps up.

Reply:
I can change the alert to be absolute and than the page won't jump, 
although it will hide the page header on smaller screens or when there's a lot of text in the alert.
Not sure what is the best option, @lizsurette we need your help here.

**Alert's position is absolute**
![a3da674a-8920-45a5-y214-b252df854615](https://user-images.githubusercontent.com/26363699/40785861-4d41d460-64f2-11e8-96e0-cf28d8c6ef5d.gif)

**Alert's display is block**
![15cb04bf-1067-42a0-yadc-d4b48a205095](https://user-images.githubusercontent.com/26363699/40786410-a6b73d2c-64f3-11e8-95a9-cb0c05f49683.gif)